### DISCUSSION TITLE
RTSP on zerotier VPN

### DISCUSSION THREAD

**Discussion by JulLEEZ** [bluenviron/mediamtx#1263] [Questions]
I have recently tried installing RTSP server on a device running in Zerotier virtual network. It seems that zerotier blocks some ports and protocols. I guess there's a way to solve it by writing some rules and adding exceptions for some ports and protocols. Has anyone tried it? Could you please tell me what ports and protocols I should add to exception rules to make RTSP server work? 
Thanks!

**Comment by aler9** [comment]
Hello, you can force all communications through port 8554 by disabling the UDP protocol with this server configuration:

```yml
protocols: [tcp]
```

Then you can publish to `rtsp://zerotier-ip:8554/mypath`.

[a comment by an automated account is not shown]
